### PR TITLE
chore(ci): create workflow to release tfhe-cuda-backend crate

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: .github/workflows/make_release_cuda.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        type: boolean
+        default: true
+      push_web_package:
+        description: "Push web js package"
+        type: boolean
+        default: true
+      push_node_package:
+        description: "Push node js package"
+        type: boolean
+        default: true
```
Comment (IceTDrinker) [inline .github/workflows/make_release_cuda.yml:22]: not required for the tfhe backend

File: .github/workflows/make_release_cuda.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    steps:
+      - name: Start instance
+        id: start-instance
+        uses: zama-ai/slab-github-runner@8562abbdc96b3619bd5debe1fb934db298f9a044
+        with:
+          mode: start
+          github-token: ${{ secrets.SLAB_ACTION_TOKEN }}
+          slab-url: ${{ secrets.SLAB_BASE_URL }}
+          job-secret: ${{ secrets.JOB_SECRET }}
+          profile: gpu-test
```
Comment (IceTDrinker) [inline .github/workflows/make_release_cuda.yml:48]: does it mean it will start a p3.2xlarge ?

Comment (soonum) [reply .github/workflows/make_release_cuda.yml:48]: yes sir

Comment (soonum) [reply .github/workflows/make_release_cuda.yml:40]: So that we are sure to build it in the right condition.

Comment (IceTDrinker) [reply .github/workflows/make_release_cuda.yml:40]: no shortage on those instances ?

**Review by IceTDrinker — APPROVED** [review]
thanks !

File: .github/workflows/make_release_cuda.yml (hunk 2 above)
Comment (soonum) [reply .github/workflows/make_release_cuda.yml:40]: Not anymore on this one. Plus this is a release workflow that is supposed to be run once every quarter.

Comment (IceTDrinker) [reply .github/workflows/make_release_cuda.yml:40]: I know, hence why it's important there are no shortages :)

**Pull request merged**